### PR TITLE
NEW: setup filter and intercative logger to listen only to Resolution…

### PR DESCRIPTION
**Pull request by bugant**
…Proposed events

### CODE REVIEW

File: issuehunter_observer.go [hunk 1]
```diff
@@ -1,67 +1,101 @@
...
 
-    "github.com/issuehunter/ethrpc"
+	"github.com/issuehunter/ethrpc"
 )
 
-func CreateFilter(ethRpcClient *ethrpc.EthRPC, contractAddress string) string {
-	log.Printf("creating a new event filter for the IssueHunter contract deployed at address %v", contractAddress)
-	var filter = map[string]interface{}{"address": contractAddress}
-	filterId, err := ethRpcClient.EthNewFilter(filter)
+func _CreateFilter(ethRpcClient *ethrpc.EthRPC, contractAddress string, filter map[string]interface{}) string {
```
Comment (potomak) [inline issuehunter_observer.go]: Private functions start with a lower case letter.

File: issuehunter_observer.go [hunk 2]
```diff
@@ -1,67 +1,101 @@
...
 	return uninstall
 }
 
+func CreateResolutionProposedFilter(ethRpcClient *ethrpc.EthRPC, contractAddress string) string {
+	log.Printf("creating a new filter on ResolutionProposed event for the IssueHunter contract deployed at address %v", contractAddress)
+	// > web3.sha3('ResolutionProposed(bytes32,address,bytes32)')
+	// "0x3b2c9742ac31922699b361e5e2c3d23b7762a77db8d2ae4b5d8004c904e9a4b7"
+	var filter = map[string]interface{}{
+		"address": contractAddress,
+		"topics":  []interface{}{"0x3b2c9742ac31922699b361e5e2c3d23b7762a77db8d2ae4b5d8004c904e9a4b7"},
```
Comment (potomak) [inline issuehunter_observer.go:62]: Not required, but we could compute this by using https://godoc.org/golang.org/x/crypto/sha3.

File: issuehunter_observer.go [hunk 3]
```diff
@@ -1,67 +1,101 @@
...
+	for {
+		GetEvents(rpcClient, filterID)
+		time.Sleep(2 * time.Second)
+	}
+}
+
+func InteractiveResolutionProposedLogObserver(url string, contractAddress string) {
+	rpcClient := ethrpc.NewEthRPC(url)
+	filterID := CreateResolutionProposedFilter(rpcClient, contractAddress)
 	c := make(chan os.Signal, 1)
```
Comment (potomak) [inline issuehunter_observer.go:89]: This is an unbuffered channel to handle OS signals, correct?

File: issuehunter_observer.go [hunk 4]
```diff
@@ -1,67 +1,101 @@
...
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 	go func() {
-		<- c
+		<-c
 		fmt.Println("quitting...")
-		UninstallFilter(rpcClient, filterId)
+		UninstallFilter(rpcClient, filterID)
 		os.Exit(1)
```
Comment (potomak) [inline issuehunter_observer.go]: Why exiting with a non-success code?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Interactive logs
+
+
+There are two utility functions in the `patchverifier` package that lets you
+inspect live logs from the node. You can use them as in the following examples:
+
+First of all start a node locally. In the following example we are connecting to the rinkeby.io test network:
+
+    geth --networkid=4 --datadir=$HOME/.rinkeby --cache=512 --ethstats='yournode:Respect my authoritah!@stats.rinkeby.io' --bootnodes=enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303 --rpc
```
Comment (potomak) [inline README.md:15]: Can I use the node that's been started by Mist?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+    package main
+      
+    import (
+        "github.com/issuehunter/patch-verifier"
+    )
+    
+    func main() {
+        patchverifier.InteractiveLogObserver("http://127.0.0.1:8545", "CONTRACT_ADDRESS_HERE")
+    }
```
Comment (potomak) [inline README.md]: I think GitHub supports Go syntax highlighting :trollface:

**Review by potomak — APPROVED** [review]
All my comments are minor issues that are not blocking this PR from being merged. 👍

File: issuehunter_observer.go (hunk 4 above)
Comment (bugant) [reply issuehunter_observer.go]: Ah right, will fix it.

File: README.md (hunk 5 above)
Comment (bugant) [reply README.md:15]: Yes sure, I think it also accept RPC call, right?

Comment (potomak) [reply README.md:15]: I guess so, that's the way I would expect Mist (the desktop app) to interact with the node.

**Pull request merged**